XenAdmin/MainWindow.cs

```diff
@@ -1025,19 +1025,18 @@ private void connection_CachePopulated(IXenConnection connection)
             if(licenseTimer != null)
                 licenseTimer.CheckActiveServerLicense(connection, false);
 
-            if ((Registry.GetBrandOverride() == "XenCenter" || BrandManager.BrandConsole == "XenCenter") &&
-                !Helpers.Post82X(connection))
+            if (BrandManager.BrandConsole == "[XenCenter]" || BrandManager.BrandConsole == "XenCenter")
             {
                 Pool pool = Helpers.GetPoolOfOne(connection);
                 if (pool != null && pool.GetHealthCheckStatus() == Pool.HealthCheckStatus.Enabled)
                 {
                     Program.BeginInvoke(Program.MainWindow, () =>
                     {
-                        using (var dlg = new WarningDialog(string.Format(Messages.PROBLEM_HEALTH_CHECK_ON_CONNECTION, pool.Name()),
-                                   new ThreeButtonDialog.TBDButton(Messages.PROBLEM_HEALTH_CHECK_HELP, DialogResult.Yes),
-                                   new ThreeButtonDialog.TBDButton(Messages.CANCEL, DialogResult.No)))
+                        using (var dlg = new InformationDialog(
+                                   string.Format(Messages.PROBLEM_HEALTH_CHECK_ON_CONNECTION, pool.Name(), BrandManager.BrandConsole),
+                                   ThreeButtonDialog.ButtonOK))
                         {
-                            if (dlg.ShowDialog() == DialogResult.Yes)
+                            if (dlg.ShowDialog() == DialogResult.OK)

```

Comment:
nit, but I think worth addressing: Users can still use the `x` button on the dialog to close it. I think some may expect it to work in the same way as a "Cancel" button. Can we remove it? Or maybe, we should remove enrollment as soon as the pool connects, and then tell the user we just did.

Reply:
We can't remove enrollment silently - for one thing if the user does not have enough permissions the operation will fail and they will get an RBAC error out of nowhere. I don't think it's a good idea in terms of UX to remove the X button either. If the changed text does not make things clearer (due to the finality in the statement) maybe we should restore the Cancel button.

Reply:
I think the bit `XC will disable Health Check and clear the enrollment before you can proceed to manage your system` does do the trick, because it doesn't sound like the user has an option which could be activated if they hit Esc, the latter rather dismisses the dialog and XC proceeds to do what it said it will.